2_process/src/process_data.R

```diff
@@ -26,13 +18,15 @@ process_data <- function() {
       model_type == 'pgdl' ~ 23
     ), n_prof = as.numeric(str_extract(exper_id, '[0-9]+')))
   # Save the processed data
-  readr::write_csv(eval_data, file = '2_process/out/model_summary_results.csv')
+  output_file = '2_process/out/model_summary_results.csv'
+  readr::write_csv(eval_data, file = output_file)
+  return(output_file)
 }
 
   
 # Save the model diagnostics
-create_diagnostic_text = function() {
-  eval_data <- read_eval_data()
+create_diagnostic_text = function(mendota_file = "1_fetch/out/model_RMSes.csv") {
```

Comment:
Similar to above -- let's add in the output filepath as an argument to this function.

Reply:
Done!